wix/yoshi
# change the public path of client applications to be static (on build time) instead of dynamic (on run time)

### 🔦 Summary
#### GA'ables
This PR sets the public path according to CI environments params (`ARTIFACT_ID`, `ARTIFACT_VERSION`) to create the proper path on parastorage.

This should create a different public path for each build, according to the app itself, so all the chunks are relative to the main chunk.

* It means that when the base statics domain of `https://static.parastorage.com` is changed, we need to create new bundles as this address will be hard coded in the bundles.

#### unpkg

~~If `unpkg: true`is set on `package.json` the public path will be constructed to work with unpkg (which means it won't work with normal `parastorage/services`)~~

### 🗺️ Test Plan
* Verify that the public path is `"/"` when these params are not supplied.
* Verify that the public path constructed well when the params supplied.
* ~~Verify that the public path constructed well when the params supplied and unpkg is set on package.json.~~
